diff -burNp maxima-5.17.0/doc/info/include-maxima.texi.in maxima-5.17.0-dud/doc/info/include-maxima.texi.in
--- maxima-5.17.0/doc/info/include-maxima.texi.in	2008-12-03 05:57:01.000000000 +0100
+++ maxima-5.17.0-dud/doc/info/include-maxima.texi.in	2009-02-17 20:15:58.766974769 +0100
@@ -21,9 +21,10 @@ This is a Texinfo Maxima Manual
 Copyright 1994,2001 William F. Schelter
 
 @format
-START-INFO-DIR-ENTRY
-* Maxima: (maxima).     A computer algebra system.
-END-INFO-DIR-ENTRY
+@dircategory Scientific software:
+@direntry
+* Maxima: (maxima).                    A computer algebra system
+@end direntry
 @end format
 @end ifinfo
 
