diff -burNp maxima-5.17.0/common.mk maxima-5.17.0-dud/common.mk
--- maxima-5.17.0/common.mk	2004-11-13 22:24:17.000000000 +0100
+++ maxima-5.17.0-dud/common.mk	2009-02-17 20:22:20.950173231 +0100
@@ -3,13 +3,13 @@
 verpkglibdir = $(pkglibdir)/@VERSION@
 verpkglibexecdir = $(libexecdir)/@PACKAGE@/@VERSION@
 verpkgdatadir = $(pkgdatadir)/@VERSION@
-docdir = $(verpkgdatadir)/doc
+docdir = $(datadir)/doc/@PACKAGE@-doc-@VERSION@
 dochtmldir = $(docdir)/html
 docsharedir = $(docdir)/share
 demodir = $(verpkgdatadir)/demo
 emacsdir = $(verpkgdatadir)/emacs
 sharedir = $(verpkgdatadir)/share
-instsrcdir = $(verpkgdatadir)/src
+instsrcdir = $(prefix)/src/@PACKAGE@-@VERSION@
 xmaximadir = $(verpkgdatadir)/xmaxima
 insttestsdir = $(verpkgdatadir)/tests
 
