--- maxima-5.41.0/src/cl-info.lisp.orig	2017-12-05 18:40:03.114984028 +0100
+++ maxima-5.41.0/src/cl-info.lisp	2017-12-05 20:06:02.988258446 +0100
@@ -207,15 +207,22 @@
      (char-count (caddr value))
      (text (make-string char-count))
      (path+filename (merge-pathnames (make-pathname :name filename) dir-name)))
-    (with-open-file (in path+filename :direction :input)
       (unless (plusp byte-offset)
 	;; If byte-offset isn't positive there must be some error in
 	;; the index.  Return nil and let the caller deal with it.
 	(return-from read-info-text nil))
+    (if (probe-file path+filename)
+    (with-open-file (in path+filename :direction :input)
       (file-position in byte-offset)
       (#-gcl read-sequence
        #+gcl gcl-read-sequence
        text in :start 0 :end char-count))
+    (with-open-stream (in (ext:run-shell-command (concatenate 'string "gunzip -c " path+filename ".gz") :output :stream))
+      (dotimes (ibytes byte-offset)
+	      (read-char in))
+      (#-gcl read-sequence
+       #+gcl gcl-read-sequence
+       text in :start 0 :end char-count)))
     text))
 
 #+gcl
