diff -burNp maxima-5.17.0/src/cl-info.lisp maxima-5.17.0-dud/src/cl-info.lisp
--- maxima-5.17.0/src/cl-info.lisp	2008-04-30 21:55:36.000000000 +0200
+++ maxima-5.17.0-dud/src/cl-info.lisp	2009-02-17 20:20:25.119541499 +0100
@@ -198,12 +198,18 @@
      (byte-count (caddr value))
      (text (make-string byte-count))
      (path+filename (make-pathname :directory dir-name :name filename)))
+    (if (probe-file path+filename)
     (with-open-file (in path+filename :direction :input)
       (file-position in byte-offset)
       #+gcl (gcl-read-sequence text in :start 0 :end byte-count)
       #-gcl (read-sequence text in :start 0 :end byte-count))
+       (with-open-stream (in (ext:run-shell-command (concatenate 'string "gunzip -c " path+filename ".gz") :output :stream))
+               (dotimes (ibytes byte-offset)
+                       (read-char in))
+               #+gcl (gcl-read-sequence text in :start 0 :end byte-count)
+               #-gcl (read-sequence text in :start 0 :end byte-count)))
     text))
 
 #+gcl
